e2e/tests/components/input-dropdown/standard/input-dropdown.po.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { browser, by, element, ElementFinder } from 'protractor';
+
+export class InputDropdownPage {
+
+    async getPage(): Promise<void> {
+        await browser.get('#/input-dropdown');
+    }
+
+    dropdown = element(by.id('dropdown'));
+    checkboxDisabled = element(by.id('disabledBtn'));
+
+    async isDropdownExpanded(): Promise<boolean> {
+        return this.dropdown.$('dropdown').isPresent();
+    }
+
+    getButton(): ElementFinder {
+        return this.dropdown.$('button.form-control');
+    }
+
+    async getButtonText(): Promise<string> {
+        return (await this.getButton()).getAttribute('innerText');
```

Comment:
`getButton` isn't marked as async so you shouldn't await it. Also, there's a `getText` function you should use instead of this.